pkg/cmd/server/start/admission.go

```diff
@@ -41,6 +41,7 @@ var (
 
 func init() {
 	defaultOffPlugins.Insert("AlwaysPullImages")
+	defaultOffPlugins.Insert("JenkinsBootstrapper")
```

Comment:
Why don't you want to remove the enabled flag?  what's the difference between enabling this feature via the admission controller's config, and enabling it by enabling the admission controller itself?

Isn't enabling/disabling the controller what @deads2k's suggestion (which i've accepted) allows for?